eviloauth/dispatcher.py

```diff
@@ -1,11 +1,80 @@
 import sys
 import logging
+from enum import Enum, auto
+from diskcache import Cache
 from eviloauth.idp import IDP
 from eviloauth.exceptions import EviloauthCommandException
+from eviloauth.access_token import AccessToken
+from eviloauth import app, cache, temp_dir, load_modules
+from datetime import datetime, timedelta
+from eviloauth.general_token import GeneralToken
+
+
+class CommandType(Enum):
```

Comment:
Find a way to build these commands dynamically, we need one source of truth for all possible commands/sub commands. Which is currently in __init__.py. If you have different sources in different places, it will diverge and cause bugs.